package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "producthighlights",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gofynd/fdk-client-javascript": "^1.0.0",
+    "@gofynd/nitrozen-react": "^2.0.2",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.2.1",
+    "bootstrap": "^5.2.3",
+    "convict": "^6.2.3",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "fdk-extension-javascript": "git+https://github.com/gofynd/fdk-extension-javascript.git#v0.5.3",
+    "ioredis": "^5.2.2",
+    "mongoose": "^7.0.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.6.1",
```

Comment:
differentiate frontend dependencies and backend dependencies in different package.json file